### PR TITLE
cargo clippy --fix

### PR DESCRIPTION
**Pull request by kylewillmon**
This was run with rustc 1.68.0

### CODE REVIEW

**Review by maxrake — DISMISSED** [review]
Automated changes LGTM.

File: cli/tests/common.rs [hunk 1]
```diff
@@ -16,18 +16,14 @@ use tempfile::TempDir;
 pub const API_URL: &str = "https://api.staging.phylum.io";
 const PROJECT_NAME: &str = "integration-tests";
 
+#[derive(Default)]
 enum Cwd {
     Path(PathBuf),
     TempDir,
+    #[default]
```
Comment (cd-work) [inline cli/tests/common.rs:23]: Isn't this a 1.65.0 feature? Our CLI is 1.64.0+ min version.

Though I guess that's fine for tests.

Comment (kylewillmon) [reply cli/tests/common.rs:23]: Good catch. Are you opposed to bumping the MSRV to 1.65?

Comment (cd-work) [reply cli/tests/common.rs:23]: Not at all.

File: cli/Cargo.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@ version = "4.7.0"
 authors = ["Phylum, Inc. <engineering@phylum.io>"]
 license = "GPL-3.0-or-later"
 edition = "2021"
-rust-version = "1.64.0"
+rust-version = "1.65.0"
```
Comment (maxrake) [inline cli/Cargo.toml:7]: Is it okay to have different MSRV values throughout the repo? With this change, there will be three different values for `rust-version` between the four (4) `Cargo.toml` files in the `cli` repository.

**Review by maxrake — APPROVED** [review]

**Review by cd-work — APPROVED** [review]

**Pull request merged**